### DISCUSSION TITLE
Sec Onion Master don't get Logs from Wazuh Agents

### DISCUSSION THREAD

**Discussion by admarnelson** [Security-Onion-Solutions/securityonion#7189] [Ideas] [answered]
Help me understand some point here: Sec Onion 2.3.70 worked perfectly I updated to 2.3.91 begins work bad now I did new update to 2.3.100 I can't get event logs from wazuh agent installed on windows and no any logs host or network
I installed wazuh agent the same version as Master.  an  ideia please... 

![image](https://user-images.githubusercontent.com/93060289/153672904-86bb8362-f59c-445a-a951-456085a8669f.png)

![image](https://user-images.githubusercontent.com/93060289/153673301-c9e08733-14a0-4e7e-9476-0bdce85d0cca.png)

![image](https://user-images.githubusercontent.com/93060289/153673860-d9709579-fedc-468d-ada1-46e4ea070196.png)

**Comment by dougburks** [comment]
First, please edit the Title of this discussion and change it from `Help me !!!!!` to something that is search friendly, relevant to the discussion, and has fewer exclamation points.

If you're having problems with the Wazuh agent, I'd recommend checking the Wazuh agent logs for clues.

**Reply by admarnelson** [reply]
Thanks for view

**Reply by dougburks** [reply]
Again, please edit the Title of this discussion to something that is search friendly and relevant to the discussion. `I need help` is generic and could apply to any discussion. The Title should be specific to this discussion.

And again I'll repeat my advice from my previous reply: if you're having problems with the Wazuh agent, I'd recommend checking the Wazuh agent logs for clues.

**Reply by admarnelson** [reply]
New instalation 2.3.100 standlone and wazuh agent installed but no logs coming to the Master. what is the problem?

**Reply by admarnelson** [reply]
I check to menu view log on agent wazuh. logs are sending

**Reply by dougburks** [reply]
Have you run `so-allow` to allow the traffic through the host firewall?

Are there any network firewalls that could be blocking the traffic?

Have you run `tcpdump` to confirm that the Wazuh agent traffic is reaching the management interface?

**Reply by admarnelson** [reply]
I decided reinstalled hold version 2.3.70 standlone. its working. I don't understand.

I follow this link here: https://noctedefensor.com/security-onion-wazuh-installation/

**Reply by admarnelson** [reply]
Anyway. 2.3.7 not bad. I will keep this version for while.
Thanks @dougburks

**Comment by admarnelson** [comment]
I would like make the SecurityOnion as my company SIEM.

**Comment by dougburks** [comment, marked as answer]
I just did a fresh installation of Security Onion 2.3.100 and then installed the Wazuh agent and sysmon on a Windows box and verified that Wazuh and sysmon logs are flowing properly:

![image](https://user-images.githubusercontent.com/1659467/154744697-eb06a75a-59f4-424c-9c94-8ef2a3b21540.png)

![image](https://user-images.githubusercontent.com/1659467/154744470-d27d4a89-08be-4c0a-8210-fd2d0da74618.png)

**Reply by admarnelson** [reply]
I would like get direct oficial contact with you @dougburks there are some questions to make (business)
If you don't mind to provide your contacts

**Reply by dougburks** [reply]
For business inquiries, please go to https://securityonionsolutions.com/, select the product or service that you're interested in, and use the contact form on that page to get in touch with the appropriate team. Thanks!
